packages/__docs__/buildScripts/DataTypes.ts

```diff
@@ -0,0 +1,283 @@
+/*
```

Comment:
this are the data types for the single pages and the metadata JSON; this is shared between the build scripts and the app

Reply:
yep, I'll make a ticket for this.